### DISCUSSION TITLE
Try syncthing core 1.18.4 on TestFlight

### DISCUSSION THREAD

**Discussion by SimonPickup** [MobiusSync/MobiusSync#56] [Beta]
We know it's been a long time between drinks here, and we're nearly ready to release a version based on Syncthing 1.18.x core to TestFlight users for beta testing. You can sign up now at https://testflight.apple.com/join/n5A7ceuK

Please note this is **beta** software and is not production ready.

**You should only use the software in an environment where you can tolerate data corruption or data loss.**

Follow this thread for further updates when the beta version(s) are distributed.

**Comment by rasa** [comment]
I tried, but I get the message
```
This beta isn't accepting any new testers right now.
```

**Reply by SimonPickup** [reply]
Hmm sorry, maybe it won't work until there's a build ready to be distributed. I was hoping to have people signed up ready in advance to get it out quickly. Will update further here.

**Comment by SimonPickup** [comment]
TestFlight now has v1.18.3 (build 202110261137) available. Please try the above sign-up link again.

Basic encrypted folder functionality seems to work OK.

**Reminder: You should only use this software in an environment where you can tolerate data corruption or data loss.**

**Comment by even-allmighty** [comment]
I just wanted to give feedback, that synching with an encrypted node its working flawless for me. I've changed an existing node to an encrypted one and its happily syncing between 1 iOS and 3 other devices.

**Reply by SimonPickup** [reply]
Thanks so much.

**Comment by SimonPickup** [comment]
I received this feedback via TestFlight. Sharing it here for others to comment and to inspire others to share their testing results too.

**Since there's no analytics in Mobius Sync, we have no other way of knowing whether the beta is successful other than your comments, so please do share on here!**

>  I installed and tested `v1.18.3-23-g6efc5a05a-dirty-i, iOS (64-bit ARM)` on iOS 14.7.1 on an iPad. My old data remained, and everything just worked, there were no errors or warnings in the log. Tested with multiple linux, Windows, Android, and iOS 1.13 devices. I did not test add/update/delete of devices or folders, encryption functionality, or a MacOS device. I'll perform more complete testing on iOS 14.8 and 15.1 and report again when done in a few days. What's your timeframe for release? One thought is maybe disabling "External File Versioning" as it wouldn't work on iOS, as each program is in it's own sandbox. I'm assuming you want beta feedback via TestFlight, and not via GitHub. Lemme know if you'd prefer the latter.

**Reply by SimonPickup** [reply]
Created (and fixed) #58

**Comment by SimonPickup** [comment]
A new TestFlight build 202110311729 has been pushed out with a fix for #26.

Syncthing v1.18.4 is expected 2021-11-02. **We hope to release Mobius Sync beta the same day, aiming at full App Store release a week later.**

**Comment by SimonPickup** [comment]
Testflight 202111040534 build is out:
- #49
- #58

Plan is to release this publicly on 9-Nov-2011, so please share your feedback here.

**Comment by SimonPickup** [comment]
Version 1.18.4 has now been published to the App Store. It seems to take up to 24 hours to propagate globally.
